node/engine/policy/policy.go

```diff
@@ -0,0 +1,37 @@
+package policy
+
+import (
+	"fmt"
+	"log/slog"
+
+	"github.com/statechannels/go-nitro/internal/logging"
+	"github.com/statechannels/go-nitro/protocols"
+)
+
+// PolicyMaker is used to decide whether to approve or reject an objective
+type PolicyMaker interface {
```

Comment:
Do we know why we chose the name `PolicyMaker`? Why not just `Policy`?

Reply:
I suspect this was based on the naming convention that [interfaces should be an "agent" noun ending with the er suffix.](https://go.dev/doc/effective_go#interface-names)